src/core/net/ip6.cpp

```diff
@@ -1448,7 +1448,8 @@ Error Ip6::RouteLookup(const Address &aSource, const Address &aDestination) cons
 #if OPENTHREAD_CONFIG_IP6_BR_COUNTERS_ENABLE
 void Ip6::UpdateBorderRoutingCounters(const Header &aHeader, uint16_t aMessageLength, bool aIsInbound)
 {
-    otPacketsAndBytes *counter = nullptr;
+    otPacketsAndBytes       *counter       = nullptr;
+    static constexpr uint8_t kPrefixLength = 48;
```

Comment:
Move constant to top (before variable definition)?

Reply:
Thanks! Done.